security/advancedtls/advancedtls.go

```diff
@@ -35,10 +35,10 @@ import (
 	credinternal "google.golang.org/grpc/internal/credentials"
 )
 
-// VerificationFuncParams contains parameters available to users when
-// implementing CustomVerificationFunc.
+// HandshakeVerificationInfo contains parameters available to users when
+// implementing PostHandshakeVerificationFunc.
```

Comment:
Similarly, should this say "contains information about a handshake needed for verification" or something?

Reply:
Changed